solidus_dev_support.gemspec

```diff
@@ -40,7 +40,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'github_changelog_generator', '~> 1.15'
   spec.add_dependency 'puma', '>= 4.3', '< 7.0'
   spec.add_dependency 'rspec_junit_formatter'
-  spec.add_dependency 'rspec-rails', '>= 4.0.0.beta3', '< 6.0'
+  spec.add_dependency 'rspec-rails', '>= 4.0.0.beta3', '< 7.0'
```

Comment:
We should also bump the minimum rspec-rails version to v5, because [this is the version for rails 5.2 and 6.x](https://github.com/rspec/rspec-rails#rspec-rails--) and [solidus 3.0 has a minimum rails version of 5.2](https://github.com/solidusio/solidus/blob/v3.0/core/solidus_core.gemspec#L28C54-L28C54)

Reply:
Done, thank you for the thorough review :)